applications/DEMApplication/custom_utilities/calculate_global_physical_properties.h

```diff
@@ -406,14 +406,14 @@ class SphericElementGlobalPhysicsCalculator
 
       array_1d<double, 3> CalculateTotalMomentum(ModelPart& r_model_part)
       {
-          OpenMPUtils::CreatePartition(OpenMPUtils::GetNumThreads(), r_model_part.GetCommunicator().LocalMesh().Elements().size(), mElementsPartition);
+          OpenMPUtils::CreatePartition(ParallelUtilities::GetNumThreads(), r_model_part.GetCommunicator().LocalMesh().Elements().size(), mElementsPartition);
 
           double m_x = 0.0;
           double m_y = 0.0;
           double m_z = 0.0;
 
           #pragma omp parallel for reduction(+ : m_x, m_y, m_z)
```

Comment:
just to say that those loops can be rewritten with the ParallelUtilities, then you don't need the manual partitioning any more

but this is certainly a different PR :)

Reply:
Sure! I am aware of that. I did not even like this old Kratos style. This change will come in a different PR, eventually. Thanks for pointing out though.